src/js/view/controls/components/menu/settings-menu.js

```diff
@@ -314,6 +314,12 @@ class SettingsMenu extends Menu {
             gearButton.setAttribute('aria-expanded', true);
         }
 
+        if (document.querySelector('.jw-breakpoint-0') || document.querySelector('.jw-breakpoint-1')) {
```

Comment:
There can be multiple players on a page, instead use `getBreakpoint(model.get('containerWidth')`

It's a util in breakpoint.ts

Reply:
Awesome, this is great to know about!